transmart-core-db/grails-app/domain/org/transmartproject/db/i2b2data/PatientDimension.groovy

```diff
@@ -35,6 +35,7 @@ class PatientDimension implements Patient {
     // private
     String sourcesystemCd
     String sexCd
+    String inTrialId
```

Comment:
When I tested this, I got proper result for inTrialId field in JSON response, so it should not be a problem. Without adding this that line:
https://github.com/thehyve/transmart-upgrade/blob/e9df81ad5d721b3c29403cfc8ae60b1c4632fda6/transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/DimensionMetadata.groovy#L157
is failing